src/generated/resources/data/occultism/recipe/ritual/misc_miner_ancient_eldritch.json

```diff
@@ -37,7 +37,7 @@
   },
   "ritual_dummy": {
     "count": 1,
-    "id": "occultism:ritual_dummy/craft_miner_ancient_eldritch"
```

Comment:
Please be mindful of renaming ids... this breaks things in packs where quests and other things may be relying on them. There's no need to be doing this in release, it should be saved for major version changes instead.

Reply:
the id defines the automatic generation of the icon, which indicates the pentacle, in this case it was necessary to change to correct it because it was visually misleading the players

Reply:
Fair. I've just had to make changes to quests after every one of these updates of yours and they haven't always been due to that sort of necessity.

Just asking that you be mindful.